src/validation/index.js

```diff
@@ -45,4 +45,23 @@ export default {
 
     return next();
   },
+
+  rememberMe: (req, res, next) => {
+
+    const errors = [];
+    const { rememberMe } = req.query;
+
+    if (rememberMe.toLowerCase() !== 'yes') {
+      errors.push(`user profile will not be auto retrieved next time. Choose "YES" to do so.`)
+    }
+
+    if (errors.length) {
```

Comment:
There is only one condition that may lead to an error, do you consider this more readable than just handling the error in the original condition that checks if `rememberMe` is `yes`?